scripts/install.sh

```diff
@@ -206,6 +211,21 @@ while [ "$1" != "" ]; do
         -r | --region )                 shift
                                         region=$1
                                         ;;
+        -llt | --lambda_latency_threshold )    shift
```

Comment:
Single dash options like you are using here are by far and away only used with single characters.  When they are used with more characters (such as the unix CLI command `tar -xvf` for example), each character stands for an individual option.  So, I would suggest just getting rid of the single dash options altogether in this case, and just keeping the double-dash longer ones.